packages/api-i18n/src/graphql/crud/locales.crud.ts

```diff
@@ -40,13 +40,21 @@ export default (context: Context<DbContext, TenancyContext, I18NContext>) => {
             return locales;
         },
         create(data) {
+            const identity = security.getIdentity();
+
             return db.create({
                 ...dbArgs,
                 data: {
                     PK: PK_LOCALE(),
                     SK: data.code,
                     code: data.code,
-                    default: data.default
+                    default: data.default,
+                    createdOn: new Date().toISOString(),
+                    createdBy: {
+                        id: identity.id,
+                        displayName: identity.displayName,
```

Comment:
Nice.

Reply:
BTW where is the code that sets the previous default locale as a non-default one? Or maybe, where is the code that sets this one as the default one? 

🤔

Reply:
I just saw it's in the GQL resolver function. If you'll have the time, maybe we could migrate those updateDefault calls there into this file (like all other apps have it).